samples/AspNetCore3xODataSample.Web/Controllers/CustomersController.cs

```diff
@@ -10,6 +10,49 @@
 
 namespace AspNetCore3xODataSample.Web.Controllers
 {
+    public class ProjectsController : ODataController
+    {
+        /*
+         * It works for request like:
+         * POST http://localhost:5000/prefix/Projects('abc')/UpdateProjectTasks
+         * Content-Type: application/json
+         * Body:
+{
+  "projectTasks":[
+    {"@odata.type":"#AspNetCore3xODataSample.Web.Models.ProjectTask","Id":"2","ProjectId":"22"},
+    {"@odata.type":"#AspNetCore3xODataSample.Web.Models.ProjectTask","Id":"3","ProjectId":"33"}
+  ]
+}
+         * 
+         * 
+         * */
+        [HttpPost]
+        public IActionResult UpdateProjectTasks([FromODataUri] string key,
+            [FromBody] ODataActionParameters parameters)
+        {
+            var projectTasks = ((IEnumerable<ProjectTask>)parameters["projectTasks"]).ToList();
+
+            var tasks = new ProjectTask[]
+            {
+            new ProjectTask { Id = "1", ProjectId = "11",
+                ProjectTaskAssignments = new ProjectTaskAssignment[]
+                {
+                    new ProjectTaskAssignment { Id = "22", ProjectTaskId = "1" }
```

Comment:
> Id = "22", ProjectTaskId = "1"  [](start = 47, length = 32)

shouldn't this (and line 47) be using projectTasks from the payload? Currently the payload is read but not used.